### DISCUSSION TITLE
Save Styles List

### DISCUSSION THREAD

**Discussion by Richt12** [lllyasviel/Fooocus#1841] [Ideas]
I appreciate the innovative feature of displaying style previews upon hovering over them – it adds a visually engaging element to the experience.

Additionally, it would be incredibly beneficial if you could introduce a "Save Styles List" functionality. This feature would allow users to conveniently select and save their preferred styles, associating each with a custom name for easy retrieval. This enhancement would streamline the workflow for users like myself who frequently switch between different styles. Instead of searching for previously used styles each time, having the ability to recall them with a simple click would significantly improve efficiency and user satisfaction.

Thank you for considering this suggestion.

**Comment by lllyasviel** [comment]
Styles are sorted by use frequency and you do not need to save frequently used styles and they are always on top
